Controller/ImagineController.php

```diff
@@ -67,6 +67,8 @@ public function __construct(
      */
     public function filterAction(Request $request, $path, $filter)
     {
+        $path = '/'.ltrim($path, '/');
```

Comment:
what if the path is an id in a storage, I mean it is not real path on the filesystem but something like mongo id or so.

IN this case it will break all such storages. Maybe we have to put it only to those resolvers\loaders which rely on real filesystem, like WebPathResolver.

Also if the path contains a `/` symbol it must be urlencoded and added to the resolve path.